tests/helpers/test-scenarios.js

```diff
@@ -111,16 +148,20 @@ function createArrayProxy(items) {
 }
 
 function createPromiseArrayFunction(items) {
-  return function() {
-    const promise = new Promise((resolve) => setTimeout(() => resolve(A(items.slice())), 10));
+  return function () {
+    const promise = new Promise((resolve) =>
+      setTimeout(() => resolve(A(items.slice())), 10)
+    );
 
     return PromiseArray.create({ promise });
   };
 }
 
 function generateScenario(name, defaultOptions, initializer) {
-  return function(baseItems, options) {
-    const items = initializer ? initializer(baseItems.slice()) : baseItems.slice();
+  return function (baseItems, options) {
+    const items = initializer
+      ? initializer(baseItems.slice())
+      : baseItems.slice();
     const scenario = { items };
 
     Ember.assign(scenario, options); // eslint-disable-line ember/new-module-imports
```

Comment:
Can this be changed to object spread while you are at it? `Ember.assign` is deprecated